base/utils.py

```diff
@@ -1,9 +1,9 @@
-from wikiwho.models import Article
+# from wikiwho.models import Article
 
 
 def is_db_running():
-    try:
-        Article.objects.count()
-    except:
-        return False
-    return True
+    # try:
+    #     Article.objects.count()
+    # except:
+    #     return False
+    return False
```

Comment:
Really curious how the previous imports worked with this bug present! The model called `Article` has been commented out since 2018 after ebdec778b9c4c55d4bc2a5626b7dc048043148ea